src/lib/hello/posts.ex

```diff
@@ -44,4 +45,46 @@ defmodule Hello.Posts do
     |> Repo.get_by!(id: id)
     |> Repo.delete()
   end
+
+  @doc """
+  Gets favorites for a tweet.
+  """
+  @spec get_favorite_by_tweet!(integer()) :: integer() | nil
+  def get_favorite_by_tweet!(tweet_id) do
+    query = from(f in Favorite, where: f.tweet_id == ^tweet_id, select: count(f.id))
+
+    Repo.one(query)
```

Comment:
```suggestion
    Favorite
    |> where(tweet_id: ^tweet_id)
    |> select([t], %{count: count(t.id)})
    |> Repo.one()
```

Functionで書くようにして下さい！ほかも同様です！